cmd/talosctl/cmd/mgmt/cluster/create.go

```diff
@@ -161,7 +161,6 @@ var (
 	dockerHostIP              string
 	withInitNode              bool
 	customCNIUrl              string
-	crashdumpOnFailure        bool
```

Comment:
Isn't this an orthogonal change?

Reply:
yeh, since we always call destroy, let it save all logs